<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1523</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电子秤自动检定系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Camera_Connect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接相机</string>
    </property>
   </widget>
   <widget class="QLabel" name="Image_Show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1191</width>
      <height>751</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera_Gather">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera_Close">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Model_Load">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模型加载</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="event_message">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>90</y>
      <width>271</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>150</y>
      <width>41</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;运&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;行&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;状&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;态&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;信&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>630</y>
      <width>41</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;识&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;别&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;结&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Controller_Experiment_15kg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>15Kg检定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Controller_Connect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接控制器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>20</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_image">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>600</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Controller_Experiment_20kg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>20</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>30Kg检定</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1523</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
